code/game/gamemodes/miniantags/guardian/types/assassin.dm

```diff
@@ -21,8 +21,9 @@
 /mob/living/simple_animal/hostile/guardian/assassin/Stat()
 	..()
 	if(statpanel("Status"))
-		if(stealthcooldown >= world.time)
-			stat(null, "Stealth Cooldown Remaining: [max(round((stealthcooldown - world.time)*0.1, 0.1), 0)] seconds")
+		var/time_left = round(COOLDOWN_TIMELEFT(src, stealth_cooldown) / 10)
```

Comment:
Can this go below 0? If so, `if(time_left)` can be truthy even if you don't intend it to and thus show up on the stat panel

Reply:
The `COOLDOWN_TIMELEFT()` define returns a minimum of 0, so it'll either be running or finished for all of these.
```dm
#define COOLDOWN_TIMELEFT(cd_source, cd_index) max(0, cd_source.cd_index - world.time)
```